tcl/envmngt.tcl.in

```diff
@@ -1236,6 +1282,29 @@ ml <- function(...){
 }}
          }
       }
+      pwsh {
+         if {[getState is_win]} {
+            reportErrorAndExit {
+               No autoinit mode available for 'pwsh' shell on Windows}
+         }
```

Comment:
You may test coverage of this error in `testsuite/modules.00-init/120-siteconfig.exp`, within section where `TESTSUITE_ENABLE_SITECONFIG_WINDOWS_PLATFORM` variable is set